server/konnectors/vente_privee.js

```diff
@@ -212,39 +213,44 @@ function parsePage(requiredFields, bills, data, next) {
       } else if (bill.date > lastBill.date) {
         lastBill = Object.assign(bill);
       }
-      // saving current bill
-      // bills.fetched.push(bill);
+
+      if (!saveOnlyLastBill) {
+        // saving current bill
+        bills.fetched.push(bill);
+      }
     }
     return true;
   });
 
-  bills.fetched.push(lastBill);
+  if (saveOnlyLastBill) {
+    bills.fetched.push(lastBill);
+  }
 
   log.debug('bills.fetched:', bills);
 
   connector.logger.info('Successfully parsed the page, bills found:',
-  bills.fetched.length);
+    bills.fetched.length);
   return next();
 }
 
 function customFilterExisting(requiredFields, bills, data, next) {
   log.debug('customFilterExisting');
   filterExisting(log, Bill)(requiredFields, bills, data, next);
-  return next();
+  return true;
 }
 
 function customSaveDataAndFile(requiredFields, bills, data, next) {
   log.debug('customSaveDataAndFile');
   const fnsave = saveDataAndFile(
     log, Bill, fileOptions, ['vente-privee', 'facture']);
   fnsave(requiredFields, bills, data, next);
-  return next();
+  return true;
 }
 
 function buildNotifContent(requiredFields, bills, data, next) {
   log.debug('buildNotifContent');
   if (bills.filtered.length > 0) {
-    const localizationKey = 'notification bills';
+    const localizationKey = 'notification vente_privee';
```

Comment:
the notification vente_privee does not exist anymore. You hsould use the 'generic one'